admin-tools/lib/src/main/scala/com/gu/mediaservice/ImagesBatchProjection.scala

```diff
@@ -0,0 +1,40 @@
+package com.gu.mediaservice
+
+import com.gu.mediaservice.lib.config.{ServiceHosts, Services}
+import com.gu.mediaservice.model.Image
+
+import scala.concurrent.duration.Duration
+import scala.concurrent.{Await, ExecutionContext, Future}
+
+object ImagesBatchProjection {
+  def apply(apiKey: String, domainRoot: String, timeout: Duration): ImagesBatchProjection =
```

Comment:
This feels a bit redundant as it has the same constructor as the class, so `new ImagesBatchProjection(args)` has the same effect - we're trading `new` for an `apply`?

Reply:
in lambda it does not make difference so i can roll back to new